openpype/tools/ayon_loader/control.py

```diff
@@ -314,8 +315,15 @@ def get_loaded_product_ids(self):
                 containers = self._host.get_containers()
             else:
                 containers = self._host.ls()
-            repre_ids = {c.get("representation") for c in containers}
-            repre_ids.discard(None)
+            repre_ids = set()
+            for container in containers:
+                repre_id = container.get("representation")
+                try:
+                    uuid.UUID(repre_id)
+                    repre_ids.add(repre_id)
+                except ValueError:
```

Comment:
I'd probably add comment why we are bypassing `ValueError`, for future generations :)

Reply:
Sorry how does this work - does this mean this just _never_ works with OpenPype containers or what's the specific case it fails on legacy scenes? Is it that a Mongo `_id` (which is `bson.ObjectId`) is not a ` uuid.UUID()` compatible string?